middleware/auth.js

```diff
@@ -0,0 +1,20 @@
+const jwt = require('jsonwebtoken')
+const { JWT_SECRET } = process.env
+
+module.exports = (req, res, next) => {
+  const token = req.header('x-auth-token')
+  console.log(token)
```

Comment:
You don't need console.log anymore, or? :D

Reply:
don't log sensitive information like an access token, imagine someone gets access to your logs and now he has all the user tokens as well. better sage than sorry :)

Reply:
Yeah, you two are right. Forgot to delete that one.